packages/widgets/Source/Viewer/Viewer.js

```diff
@@ -402,6 +402,15 @@ function Viewer(container, options) {
   container = getElement(container);
   options = defaultValue(options, defaultValue.EMPTY_OBJECT);
 
+  //>>includeStart('debug', pragmas.debug);
```

Comment:
Should this pragma be closed around line 413?

Reply:
My bad, I forgot to close the debug pragma. Fixed it now. Thanks for catching that!